app/models/piece.rb

```diff
@@ -11,6 +11,10 @@ def user
     color == "white" ? game.white_player : game.black_player
   end
 
+  def is_move_malformed?(new_rank, new_file)
+    rank == new_rank && file == new_file
+  end
```

Comment:
It probably makes sense to have this method be protected as well, so I would move it down below the `protected` keyword. See here for more info on `public`, `private`, and `protected` in Ruby: http://stackoverflow.com/questions/3534449/why-does-ruby-have-both-private-and-protected-methods